app/controllers/publishers/accounts_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Publishers::AccountsController < ApplicationController
+  helper_method :publisher
+
+  def unsubscribe
+    publisher.update(unsubscribed_from_expired_vacancy_prompt_at: Time.zone.now)
```

Comment:
We should always use Time.current instead of Time.zone.now. And also Date.current instead of Date.today